content/testable-architecture.md

```diff
@@ -3,4 +3,62 @@ title: Testable Architecture
 description: Designing for testability
 ---
 
-Testable Architecture
\ No newline at end of file
+Automated tests and executable specifications can only work with software that has a testable architecture.
+A testable architecture enables fast and reliable tests that are easy to write, execute and maintain.
+Fast tests enable the developers to run them and obtain fast feedback on what they are building, without losing focus or flow.
```

Comment:
... enable the developers to run them and obtain ...

could use an additional word 

... enable the developers to run them continuously and obtain ...

... enable the developers to run them often and obtain ...

... enable the developers to run them frequent and obtain ...